### DISCUSSION TITLE
首页推荐应用中，明明已安装了MySQL但未识别，仍然可再次安装

### DISCUSSION THREAD

**Discussion by wwuddan123** [1Panel-dev/1Panel#4880] [Ideas]
### 1Panel 版本

v1.10.6-lts

### 问题描述

首页推荐应用中，明明已安装了MySQL但未识别，仍然可再次安装

### 重现步骤

打开首页即可

### 期待的正确结果

期望能正确识别已安装的应用，最好已安装的能变成“打开”按钮，我已经提交了功能建议的issue

### 附加信息

![image](https://github.com/1Panel-dev/1Panel/assets/27287359/85836685-e3cb-4924-a49c-965206459e2b)

**Comment by realzolo** [comment]
这不是BUG, 这是特性好吧。有些应用可以安装多个，就比如MySQL

**Comment by zhengkunwang223** [comment]
是的 有些应用可以安装多个 所以首页只是展示一下

**Comment by wwuddan123** [comment]
好的，确实可以安装多个，只是个人觉得把这个“推荐应用”放在首页没问题，但也应该有一个已安装应用的快速打开入口。

另外像OpenResty这种只能安装一个的，安装后还是一直呆在“推荐应用”第一个，又不能互动，没啥意义呀。

建议“推荐安装应用”和“快速打开应用”，这俩区域应该同时存在。像OpenResty这种已经安装过了且不能再次安装的，就要从“推荐安装应用”中移除。

**Comment by itpoo** [comment]
我觉得就目前这个没问题呀，比如我要安装不同版本的MySQL。
